Is cuda available? False
Number of different initial conditions:  16
['delta', 'delta_omega', 'delta_Id', 'delta_Id_dt'] Variables
[[-0.02, 0.02], [-0.01, 0.01], [-0.01, 0.01], [-0.1, 0.1]] Set of values for init conditions
[2, 2, 2, 2] Iterations per value
Shape: (16, 4)
üü° Solving initial condition 1/16: [-0.006625968577480556, 0.0014253276279376469, 0.00897713727909418, 0.014888319828878727]
‚ùå Error with IC 1: [-0.006625968577480556, 0.0014253276279376469, 0.00897713727909418, 0.014888319828878727], error: 'int' object is not iterable
üü° Solving initial condition 2/16: [-0.006625968577480556, 0.0014253276279376469, 0.00897713727909418, -0.06657463093673457]
‚ùå Error with IC 2: [-0.006625968577480556, 0.0014253276279376469, 0.00897713727909418, -0.06657463093673457], error: 'int' object is not iterable
üü° Solving initial condition 3/16: [-0.006625968577480556, 0.0014253276279376469, -0.004118691989227258, 0.014888319828878727]
‚ùå Error with IC 3: [-0.006625968577480556, 0.0014253276279376469, -0.004118691989227258, 0.014888319828878727], error: 'int' object is not iterable
üü° Solving initial condition 4/16: [-0.006625968577480556, 0.0014253276279376469, -0.004118691989227258, -0.06657463093673457]
‚ùå Error with IC 4: [-0.006625968577480556, 0.0014253276279376469, -0.004118691989227258, -0.06657463093673457], error: 'int' object is not iterable
üü° Solving initial condition 5/16: [-0.006625968577480556, -0.009060260666199532, 0.00897713727909418, 0.014888319828878727]
‚ùå Error with IC 5: [-0.006625968577480556, -0.009060260666199532, 0.00897713727909418, 0.014888319828878727], error: 'int' object is not iterable
üü° Solving initial condition 6/16: [-0.006625968577480556, -0.009060260666199532, 0.00897713727909418, -0.06657463093673457]
‚ùå Error with IC 6: [-0.006625968577480556, -0.009060260666199532, 0.00897713727909418, -0.06657463093673457], error: 'int' object is not iterable
üü° Solving initial condition 7/16: [-0.006625968577480556, -0.009060260666199532, -0.004118691989227258, 0.014888319828878727]
‚ùå Error with IC 7: [-0.006625968577480556, -0.009060260666199532, -0.004118691989227258, 0.014888319828878727], error: 'int' object is not iterable
üü° Solving initial condition 8/16: [-0.006625968577480556, -0.009060260666199532, -0.004118691989227258, -0.06657463093673457]
‚ùå Error with IC 8: [-0.006625968577480556, -0.009060260666199532, -0.004118691989227258, -0.06657463093673457], error: 'int' object is not iterable
üü° Solving initial condition 9/16: [0.01465242742077619, 0.0014253276279376469, 0.00897713727909418, 0.014888319828878727]
‚ùå Error with IC 9: [0.01465242742077619, 0.0014253276279376469, 0.00897713727909418, 0.014888319828878727], error: 'int' object is not iterable
üü° Solving initial condition 10/16: [0.01465242742077619, 0.0014253276279376469, 0.00897713727909418, -0.06657463093673457]
‚ùå Error with IC 10: [0.01465242742077619, 0.0014253276279376469, 0.00897713727909418, -0.06657463093673457], error: 'int' object is not iterable
üü° Solving initial condition 11/16: [0.01465242742077619, 0.0014253276279376469, -0.004118691989227258, 0.014888319828878727]
‚ùå Error with IC 11: [0.01465242742077619, 0.0014253276279376469, -0.004118691989227258, 0.014888319828878727], error: 'int' object is not iterable
üü° Solving initial condition 12/16: [0.01465242742077619, 0.0014253276279376469, -0.004118691989227258, -0.06657463093673457]
‚ùå Error with IC 12: [0.01465242742077619, 0.0014253276279376469, -0.004118691989227258, -0.06657463093673457], error: 'int' object is not iterable
üü° Solving initial condition 13/16: [0.01465242742077619, -0.009060260666199532, 0.00897713727909418, 0.014888319828878727]
‚ùå Error with IC 13: [0.01465242742077619, -0.009060260666199532, 0.00897713727909418, 0.014888319828878727], error: 'int' object is not iterable
üü° Solving initial condition 14/16: [0.01465242742077619, -0.009060260666199532, 0.00897713727909418, -0.06657463093673457]
‚ùå Error with IC 14: [0.01465242742077619, -0.009060260666199532, 0.00897713727909418, -0.06657463093673457], error: 'int' object is not iterable
üü° Solving initial condition 15/16: [0.01465242742077619, -0.009060260666199532, -0.004118691989227258, 0.014888319828878727]
‚ùå Error with IC 15: [0.01465242742077619, -0.009060260666199532, -0.004118691989227258, 0.014888319828878727], error: 'int' object is not iterable
üü° Solving initial condition 16/16: [0.01465242742077619, -0.009060260666199532, -0.004118691989227258, -0.06657463093673457]
‚ùå Error with IC 16: [0.01465242742077619, -0.009060260666199532, -0.004118691989227258, -0.06657463093673457], error: 'int' object is not iterable
/opt/anaconda3/envs/myenv/lib/python3.11/site-packages/numpy/core/fromnumeric.py:3504: RuntimeWarning: Mean of empty slice.
  return _methods._mean(a, axis=axis, dtype=dtype,
/opt/anaconda3/envs/myenv/lib/python3.11/site-packages/numpy/core/_methods.py:129: RuntimeWarning: invalid value encountered in scalar divide
  ret = ret.dtype.type(ret / rcount)
/opt/anaconda3/envs/myenv/lib/python3.11/site-packages/numpy/core/_methods.py:206: RuntimeWarning: Degrees of freedom <= 0 for slice
  ret = _var(a, axis=axis, dtype=dtype, out=out, ddof=ddof,
/opt/anaconda3/envs/myenv/lib/python3.11/site-packages/numpy/core/_methods.py:163: RuntimeWarning: invalid value encountered in divide
  arrmean = um.true_divide(arrmean, div, out=arrmean,
/opt/anaconda3/envs/myenv/lib/python3.11/site-packages/numpy/core/_methods.py:198: RuntimeWarning: invalid value encountered in scalar divide
  ret = ret.dtype.type(ret / rcount)
Mean time per iteration:  nan  and std:  nan
Time taken to solve the model for 16 initial conditions: 0.0015370845794677734 seconds.
[DEBUG] t_span = (0, 5)
Number of files in the directory:  9
Saved dataset "('GFL_4th_order', 'dataset_v10')".
